build-configs/configs/integreat-test-cms/assets/manifest.json

```diff
@@ -1 +1 @@
-../../integreat/assets/manifest.json
\ No newline at end of file
+"../../integreat/assets/manifest.json"
```

Comment:
What is this file about? I kind of think this should not exist in the first place. Is it generated by webpack?

Reply:
Hm, even if this file has been gegenerated automatically, it resist in this path and produces a failure message, that it is not a valid json file. And my IDE suggests double qoutes around the path instead of single ones --- after that I got rid of that message. I've had no further intentions with it. Let's talk about.